pkg/relayer/proxy/errors.go

```diff
@@ -3,9 +3,11 @@ package proxy
 import sdkerrors "cosmossdk.io/errors"
 
 var (
-	codespace                       = "relayer/proxy"
-	ErrUnsupportedRPCType           = sdkerrors.Register(codespace, 1, "unsupported rpc type")
-	ErrInvalidRelayRequestSignature = sdkerrors.Register(codespace, 2, "invalid relay request signature")
-	ErrInvalidSession               = sdkerrors.Register(codespace, 3, "invalid session")
-	ErrInvalidSupplier              = sdkerrors.Register(codespace, 4, "invalid supplier")
+	codespace                            = "relayer/proxy"
```

Comment:
@Olshansk is there already / have we subscribed to a convention on namespacing codespaces? I ask because I've been doing things like `codespace = "relayer_miner"`, for example. Thoughts?

Reply:
We don't have a pattern for this (yet).

Let's go with `relayer_proxy` because a `/` can be misinterpreted by someone/something.